web/cypress/integration/route/create-edit-route-with-redirect-plugin.spec.js

```diff
@@ -99,7 +98,6 @@ context('Create Edit and Delete Route with redirect plugin', () => {
     cy.contains(data.submitSuccess);
     cy.contains('Goto List').click();
     cy.url().should('contains', 'routes/list');
-    cy.contains(newName).should('be.visible');
```

Comment:
ditto, We need some assertions to be sure that the data return is normal

Reply:
when we return routes/list, because the paging, 

> Hi, @Sh1-5 Please check the logic when configuring the route, the default value of radio is not available 🤔 ![image](https://user-images.githubusercontent.com/72343596/183039537-805a6678-909b-437d-86a9-195a5107ca9a.png)

ok, I found it just now

Reply:
because paging, I can't find it.

Reply:
I don't quite understand what this pagination refers to? Is it due to old data？I honestly don't think the problem should occur.

Reply:
I use http://20.210.250.99:9000 as service, when I created a route, it was at the back of the page, the first page does't contain it

Reply:
Unless lgo search

Reply:
Unless go search